### ISSUE TITLE
New SRB plume family: Virgin Galactic

### ISSUE THREAD

**Issue by JadeOfMaar** [CobaltWolf/Bluedog-Design-Bureau#528]
Hey @jsolson I figure you'll be quite interested in this. Smoke is included. Only two of them (SL and Heavy) are trailer smoke. [Assets + readme here](https://github.com/JadeOfMaar/PlumeParty/tree/master/Engines/Solid2)
Configs for these don't exist yet but all my rules for plume combinations still apply. For the smoke, my initial suggestions for offset are as follows:
* `localPosition = 0, 0, 1.3` for `VirginVacFume`
* `localPosition = 0, 0, 1.8` for `VirginSLFume`
* `localPosition = 0, 0, 1.4` for `VirginSepFume`
* `localPosition = 0, 0, 5.5` for `VirginHeavyFume`

Feel free to use Virgin Sep for small motors other than separatrons.
![Virgin plumes](https://raw.githubusercontent.com/JadeOfMaar/PlumeParty/master/Engines/Solid2/Solids.jpg)

**Comment by JadeOfMaar** [comment]
Everything that I've documented concerning my plumes basically. Effect pairs needing identical settings, and the option to use scaling. :) 
Odd that you're not getting alerts. I'm about to post another untested thing btw. Perhaps look out for it, though I'm not in a rush for anyone to notice.

**Comment by JadeOfMaar** [comment]
cool. :) I'm looking forward to a screenshot of that.
I finally posted some "generic" flames ~~also with no config~~. I intend to maybe release those into the ~~wild~~ forum with configs for Squad's engines.

**Comment by jsolson** [comment]
It's supposed to be the "dirty" exhaust gas igniting in the main plume. I tried mixing different colored plumes together to darken it but that didn't really work as I hoped.

![untitled](https://user-images.githubusercontent.com/16825376/52164895-d62a7e80-26c6-11e9-8f92-79a4f8bb9974.png)

A pure blue version of Hypergolic/Fizzle would be nice. That's becoming my go-to plume for upper stages. BlueOrigin/hydroVac get's a little washed out, especially in clusters.

![untitled](https://user-images.githubusercontent.com/16825376/52164958-e1ca7500-26c7-11e9-98a2-90227bd75d7a.png)

**Comment by jsolson** [comment]
Thanks!

I need to fiddle a little so it's not so bright. Emission is x2 on this one.
![untitled](https://user-images.githubusercontent.com/16825376/52181835-669ab900-27c4-11e9-8c84-e8043eb50b8e.png)

**Comment by jsolson** [comment]
Still tinkering with this. Added one of the Lamp effects for the nozzle glow. I'm not sure if I'm going to keep that or not.

I had to really dial back the emission values on the fume and spark effect to minimize the wash out on clusters.

![untitled](https://user-images.githubusercontent.com/16825376/52308931-a69ca000-296c-11e9-8ddb-2238ffdf7385.png)

~~~
		running_engine
		{
			AUDIO
			{
				channel = Ship
				clip = Bluedog_DB/Sounds/KW/sound_altloop
				volume = 0.0 0.0
				volume = 0.01 0.1
				volume = 1.0 1.0
				pitch = 0.0 1
				pitch = 1.0 1
				loop = true
			}
			
			MODEL_MULTI_PARTICLE
			{
				name = lamp
				modelName = Bluedog_DB/FX/PlumeParty/Engines/BlueOrigin/hydroSLLamp
				transformName = thrustTransform
				emission = 0.0 0.0
				emission = 0.01 0.1
				emission = 1.0 1.0
				speed = 0.0 0.5
				speed = 1.0 0.5
				energy = 0.0 0.1
				energy = 1.0 0.75
				localScale = 0.2, 0.2, 0.2
				localPosition = 0, 0, 0
			}
			
			MODEL_MULTI_PARTICLE
			{
				name = fume
				modelName = Bluedog_DB/FX/PlumeParty/Engines/Hypergolic/VacFizzleFumeBlue
				transformName = thrustTransform
				emission = 0.0 0.0
				emission = 0.01 0.1
				emission = 1.0 0.16
				speed = 0.0 2.0
				speed = 1.0 2.0
				energy = 0.0 0.5
				energy = 1.0 1.0
				localScale = 1.0,1.0,1.0
				localPosition = 0,0,0.3
			}
			
			MODEL_MULTI_PARTICLE
			{
				name = spark
				modelName = Bluedog_DB/FX/PlumeParty/Engines/Hypergolic/VacFizzleSparkBlue
				transformName = thrustTransform
				emission = 0.0 0.0
				emission = 0.01 0.01
				emission = 1.0 0.03
				speed = 0.0 1.5
				speed = 1.0 1.5
				energy = 0.0 1.0
				energy = 1.0 2.0
				localScale = 0.5,0.5,0.5
				localPosition = 0,0,0.5
			}
		}
~~~

**Comment by jsolson** [comment]
@JadeOfMaar is white smoke possible? I''m still preferring the texture of fx_smokeTrail_medium for dark smoke.

![untitled](https://user-images.githubusercontent.com/16825376/52438654-cb695280-2ae7-11e9-9aac-d6babfbd4d48.png)

**Comment by JadeOfMaar** [comment]
Hey @jsolson sorry for being scarce. I dimmed the blue fizzle plumes to help with the washout and emission problems. I hope it's enough. I also just added and tested trailing smoke. I've outlined the spectrum that'll be provided too, in the readme.

> [Smoke folder](https://github.com/JadeOfMaar/PlumeParty/tree/master/Engines/Smoke)

![Smoke](https://raw.githubusercontent.com/JadeOfMaar/PlumeParty/master/Engines/Smoke/SmokeBillboard.jpg)

**Comment by jsolson** [comment]
Thanks! I'll play this week.

> The prefab vacuum cutoff switch was nver made available in KSP PartTools.

Sigh.

**Comment by JadeOfMaar** [comment]
Hey dude. I've been on-and-off making a proper BDB F-1 plume. now that I've seen the numbers you used on a Tundra plume for it I adjusted accordingly. Here are previews of it, with my normal plume export size beside it for scale.
When I can, I'll setup its behavior in-game to make it easy on you if/when you want to tweak it later.
* [Youtube](https://youtu.be/W6anDdUPmVg)
![F1 with Titan for scale](https://i.imgur.com/GLnexHK.png)

**Comment by jsolson** [comment]
Yeah I'm hearing the current F1 is a performance killer, so something sized right might be good. I think I've been going a little overboard with the emission values.

I haven't really had a lot of time to play this week.

Without changing anything the cryogenic plumes on the J2 and RL10 are flickering now. They don't really look dimmer, just a much lower framerate. I'm going to have to retry those from scratch. I need to try them with the spark effect added back in. I've got two basic variations of upper plumes going and I'm not sure which I prefer. Trying to find the right balance of hollywood/real. Real would be almost completely invisible - that's boring.

![screenshot46](https://user-images.githubusercontent.com/16825376/52890454-ca24cf00-3152-11e9-9e69-fb99e36f8a58.png)


Smoke is giving me trouble. I've got it scaled way up, and have to bump up the emission higher than I'm comfortable to fill it out. I've yet to beat fx_smoketrail_medium with it. My impression is the stock effect grows a bit more, and stays more cohesive at the tail end of the trail.

I think it needs some underside shadowing to give it shape, but I'm not sure if that would just darken it so it looks like the stock effect. Eliminating the transparency might help. It could be whiter.

![untitled](https://user-images.githubusercontent.com/16825376/52890365-63071a80-3152-11e9-874c-95031fa4b9df.png)
